packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue

```diff
@@ -16,58 +16,60 @@
 			:active="activeTab === FunmanTabs.notebook"
 			@click="activeTab = FunmanTabs.notebook"
 		/>
-		<Button
-			v-if="!showSpinner"
-			class="p-button-sm run-button"
-			label="Run"
-			icon="pi pi-play"
-			@click="runMakeQuery"
-		/>
 		<tera-progress-spinner v-if="showSpinner" :font-size="2" />
 	</div>
-	<div v-if="activeTab === FunmanTabs.wizard" class="container">
-		<div class="left-side">
+	<div v-if="activeTab === FunmanTabs.wizard" class="content">
+		<div class="container">
 			<h4 class="primary-text">Set validation parameters <i class="pi pi-info-circle" /></h4>
 			<p class="secondary-text">
 				The validator will use these parameters to execute the sanity checks.
 			</p>
-			<div class="button-row">
-				<label>Tolerance</label>
-				<InputNumber
-					mode="decimal"
-					:min-fraction-digits="0"
-					:max-fraction-digits="7"
-					v-model="tolerance"
-				/>
-			</div>
-			<div class="section-row">
-				<!-- This will definitely require a proper tool tip. -->
-				<label>Select parameters to synthesize<i class="pi pi-info-circle" /></label>
-				<div v-for="(parameter, index) of requestParameters" :key="index" class="button-row">
-					<label>{{ parameter.name }}</label>
-					<Dropdown v-model="parameter.label" :options="labelOptions"> </Dropdown>
-				</div>
-			</div>
-			<div class="section-row">
-				<div class="button-row">
+			<div class="section-row timespan">
+				<div class="button-column">
 					<label>Start time</label>
-					<InputNumber class="p-inputtext-sm" inputId="integeronly" v-model="startTime" />
+					<InputNumber inputId="integeronly" v-model="startTime" />
 				</div>
-				<div class="button-row">
+				<div class="button-column">
 					<label>End time</label>
-					<InputNumber class="p-inputtext-sm" inputId="integeronly" v-model="endTime" />
+					<InputNumber inputId="integeronly" v-model="endTime" />
 				</div>
-				<div class="button-row">
+				<div class="button-column">
 					<label>Number of steps</label>
-					<InputNumber class="p-inputtext-sm" inputId="integeronly" v-model="numberOfSteps" />
+					<InputNumber inputId="integeronly" v-model="numberOfSteps" />
 				</div>
 			</div>
 			<InputText
 				:disabled="true"
-				class="p-inputtext-sm"
+				class="p-inputtext-sm timespan-list"
 				inputId="integeronly"
 				v-model="requestStepListString"
 			/>
+			<p v-if="!showAdditionalOptions" @click="toggleAdditonalOptions" class="green-text">
+				Show additional options
+			</p>
+			<p v-if="showAdditionalOptions" @click="toggleAdditonalOptions" class="green-text">
+				Hide additional options
+			</p>
+			<div v-if="showAdditionalOptions">
+				<div class="button-column">
+					<label>Tolerance</label>
+					<InputNumber
+						mode="decimal"
+						:min-fraction-digits="0"
+						:max-fraction-digits="7"
+						v-model="tolerance"
+					/>
+				</div>
+				<Slider v-model="tolerance" :min="0" :max="1" :step="0.01" />
```

Comment:
Your input box tolerance and slider tolerance have different constraints
- input box is basically unbounded, while
- slider is between 0 and 1

What is the actual restriction on funman?

Reply:
Very good catch, was assuming between 0 and 1 based off of chats but I will need to confirm and fix one of these two inputs.

Reply:
Confirmed it should be between 0 and 1 inclusive